src/modules/auth/screens/login-screen.tsx

```diff
@@ -1,30 +1,108 @@
+import { useSignIn } from '../../../api/signin';
 import { TouchableOpacity } from 'react-native'
 import { Image, Text, View, ImageBackground, SafeAreaView, Button, TextInput } from '../../../ui/core'
 import images from '../../../ui/assets/images';
 import type { AuthStackScreenProps } from '../../../navigation/types';
+import { useEffect, useState } from 'react';
+import {
+  CardNotification,
+  hideNotification,
+  showNotification,
+} from '../../../ui/core/notifications/card-notification';
+import { signIn } from '../../../core';
+import { common } from '../../../translations/en.json'
+import { LogBox } from 'react-native';
```

Comment:
It's not a blocker, but always adds values to use path alias
https://webreaper.dev/posts/tsconfig-paths-setup/

Reply:
I will change in the next PR.